cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -0,0 +1,588 @@
+    const snapshotWidthsBubble = [151, 303];
+    const snapshotWidthsEventTile = [133, 151, 267, 303, 1024, 1920];
+
+    /**
+     * Take snapshots on modern and bubble layouts, outputting log for reference/debugging.
+     * Note that this does not take snapshot of players on IRC layout to keep the number of
+     * taking snapshots as low as possible.
+     * @param detail The Percy snapshot name. Used for outputting logs too.
+     */
+    const takeSnapshots = (detail: string) => {
+        // Check the status of the seek bar
+        // TODO: check if visible - currently visibility check on a narrow timeline causes an error
+        cy.get(".mx_AudioPlayer_seek input[type='range']").should("exist");
+
+        // Enable group layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+
+        // Click the timestamp to highlight the event tile in case it is not visible
+        cy.get(".mx_EventTile_last[data-layout='group'] .mx_MessageTimestamp").click();
+
+        // Assert that rendering of the player settled and the play button is visible before taking a snapshot
+        checkPlayerVisibility();
+
+        // Take a snapshot on group layout
+        cy.get(".mx_MAudioBody").percySnapshotElement(detail + " on group layout", {
+            percyCSS,
+            widths: snapshotWidthsGroup,
+        });
+
+        cy.log("Took a snapshot of " + detail + " on group layout");
+
+        // Enable bubble layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Bubble);
+
+        // Click the timestamp to highlight the event tile in case it is not visible after changing the layout
+        cy.get(".mx_EventTile_last[data-layout='bubble'] .mx_MessageTimestamp").click();
+
+        checkPlayerVisibility();
+
+        // Take a snapshot on bubble layout
+        cy.get(".mx_MAudioBody").percySnapshotElement(detail + " on bubble layout", {
+            percyCSS,
+            // 243px + 12px + 48px = 303px. See _EventBubbleTile.pcss and _AudioPlayer.pcss for spacing
+            widths: snapshotWidthsBubble,
+        });
+
+        cy.log("Took a snapshot of " + detail + " on bubble layout");
+
+        // Reset the layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+    };
+
+    beforeEach(() => {
+        cy.startHomeserver("default").then((data) => {
+            homeserver = data;
+            cy.initTestUser(homeserver, TEST_USER).then(() =>
+                cy.createRoom({}).then((_roomId) => {
+                    roomId = _roomId;
+                }),
+            );
+        });
+
+        cy.injectAxe();
+    });
+
+    afterEach(() => {
+        cy.stopHomeserver(homeserver);
+    });
+
+    it("should be correctly rendered on IRC layout", () => {
+        // Please note that takeSnapshots() is not to be used here since it does not take snapshots on IRC layout.
+        // The design of it should be same as on modern layout.
```

Comment:
```suggestion
        // We cannot use takeSnapshots() here since it does not take snapshots on IRC layout.
        // The design of the player on IRC layout should be same as on modern layout.
```